internals/onepassword/cli_v1.go

```diff
@@ -0,0 +1,118 @@
+package onepassword
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+)
+
+type OPV1CLI struct {
+	version string
```

Comment:
I added the version because I was worried that some small changes were made to the command signature in different versions of v1. By having the version, I could easily change the signature based on the version that brought the change. Fortunately, this is not the case.

Therefore, I can easily remove the version without worrying that some version brings a breaking change in the command signature. If this will happen, we can bring the `version` back and go from there.